functions/src/mail/sendgrid.ts

```diff
@@ -89,6 +89,8 @@ export interface user_contact {
 
 export const upsertContact = async (request: Request, response: Response): Promise<void> => {
   const data: user_contact = request.body;
+  // If list does not exist in the request, use ACM's global mailing list
+  data.list = data.list ?? "812fb281-b20b-405a-a281-097cc56210e0";
```

Comment:
@Stevenmqnguyen Shall I go ahead and add a secret for the global mailing list UID?

Reply:
Yep, use same naming convention. I'll add it into Doppler.

Reply:
Alright, let's use `MAILCHIMP_LIST_UID`. I'll make a change real quick.

Reply:
@NickBurnett It would be `SENDGRID_LIST_UID`, we stopped using mail chimp. Probably add the word global in the variable name somewhere too.

Reply:
So I actually just realized that this is a PR based on his fork so @legitatx please change the global UID to `environment.MAILCHIMP_LIST_UID` and import the environment file at the top if it isn't already.

Reply:
> So I actually just realized that this is a PR based on his fork so @legitatx please change the global UID to `environment.MAILCHIMP_LIST_UID` and import the environment file at the top if it isn't already.

environment.SENDGRID_GLOBAL_LIST_UID I've added into doppler.

Reply:
@NickBurnett @Stevenmqnguyen Just added the Doppler env variable in the latest commit.